cmd/relay-s3/README.md

```diff
@@ -0,0 +1,50 @@
+# relay-s3
+
+A drand relay that writes randomness rounds to an AWS S3 bucket.
+
+## Usage
+
+```sh
+drand-relay-s3 run [arguments...]
+```
+
+Note: at minimum you'll need to specify a S3 bucket name and either a HTTP, gRPC or libp2p pubsub drand endpoint to relay from.
+
+**Example**
+
+```sh
+drand-relay-s3 run -hash 138a324aa6540f93d0dad002aa89454b1bec2b6e948682cde6bd4db40f4b7c9b -url http://pl-us.testnet.drand.sh -bucket drand-testnet -region eu-west-2
+```
+
+### Sync bucket with randomness chain
+
```

Comment:
the `begin` flag is confusingly named / explained here.
I was not expecting it to go from that round and count down - i was expecting it to go from that round and count up instead.

Reply:
can you instead fetch a list of the rounds in the bucket already, and populate the files that are not already uploaded?

Reply:
> the `begin` flag is confusingly named / explained here.
> 
> I was not expecting it to go from that round and count down - i was expecting it to go from that round and count up instead.

I figured newer rounds were more important so should be populated first.

Reply:
> can you instead fetch a list of the rounds in the bucket already, and populate the files that are not already uploaded?

Yes, it’s a paginated API for listing objects. I did not have enough time to do it this way last night 😛

Reply:
i don't care if missing rounds are populated from big->small or small->big, but given that new rounds will accumulate over time at the end, there should be some way to periodically add the set of new rounds that haven't yet been pushed. I think that's why i was expecting some way to specify  "--greater-than <last sync point>" as a way to do that.

if `sync` can do the api listing and auto determine what needs to be pushed than we can get rid of needing a flag entirely, which also resolves the mental model issue :)

Reply:
If I remove the flag do you think this is mergable? I’ll follow up with a PR for the ideal solution when I have time.

Reply:
remove, or flip to move forward so that it can be used to fill in new un-pushed results.
(or file issues for the future work, and we can do it as is :) )

Reply:
I have switched to forwards sync - it simplifies the code a little.